host/traceh.c

```diff
@@ -314,6 +314,23 @@ oe_result_t oe_log(oe_log_level_t level, const char* fmt, ...)
     return result;
 }
 
+void* _oe_log_context = NULL;
```

Comment:
nit: global variables should be initialized already.

Reply:
why not making these variable `static`?

Reply:
At least these variable also need to be visible to host/host.c, so they are global.